nodeJs/week2/app.js

```diff
@@ -0,0 +1,58 @@
+import express from "express";
+import jsonData from "./document.json" assert { type: "json" };
+
+const app = express();
+
+const port = process.env.PORT || 3000;
+
+// Support parsing JSON requests
+app.use(express.json());
+
+app.get("/search", (req, res) => {
+  const q = req.query.q;
+  if (q) {
+    const filteredData = jsonData.filter((item) => {
+      return item.value.includes(q);
```

Comment:
since the q can be formatted in many way, maybe it would be worth to make filtering format-agnostic? :)